packages/sdk/wallets/wallet-walletconnect/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@celo/wallet-walletconnect",
+  "version": "1.0.3-dev",
```

Comment:
Are we not incrementing versions separately now? I thought with the split these would have separate version numbers.

Reply:
Not sure to be honest, all the others have 1.0.3-dev as well. @medhak1 what should I do here?

Reply:
Confirmed it should line up with the other versions